Sort-the-odd.js

```diff
@@ -0,0 +1,21 @@
+// Task
+// You will be given an array of numbers. You have to sort the odd numbers in ascending order while leaving the even numbers at their original positions.
```

Comment:
Solution does not match the task. You have 2 files with 2 solutions for the array diff task. And no solutions for order task as far as I see.